code/python/README.md

```diff
@@ -1 +1,3 @@
 The Python script chlamy_modeler.py uses matplotlib, a popular data visualization library, to create and display a graphical representation of cellular dimensions as an ellipse, based on provided mean dimensions and standard deviations. This script is specifically designed to handle cell size data, taking mean values and standard deviations for length and width to create ellipses that represent average cell size and cell size variability. The script not only generates and displays a graphical representation of a cell's dimensions (length and width), considering the mean and standard deviation of these dimensions, but also computes and displays the cell's eccentricity. The output is both a visual plot and a saved SVG file. This SVG file can be further stylized in programs like Adobe Illustrator, which was the process used to generate the model figure in this pub.
+
+The Python script focus_varLaplac.2.1.py is designed to process a collection of TIF image stacks, focusing on identifying and isolating in-focus frames within each stack. It navigates through a specified root directory and its subdirectories looking for TIF files, then applies a focus measure algorithm to each frame in these stacks to determine which are in focus. Finally, it saves the in-focus frames and their adjacent frames as new TIF files, organized by experiment and species, in an output directory.
```

Comment:
Just a broad comment which is applicable to all file names. I would follow the `_` convention for each file and not mix camelcase and snakecase. 

And I'd go for descriptive filenames over shortened names. `focus_varLaplac` doesn't mean much to me :/